twint/src/main/java/com/adyen/checkout/twint/Twint.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by oscars on 17/11/2023.
+ */
+
+package com.adyen.checkout.twint
+
+import androidx.activity.ComponentActivity
+import androidx.lifecycle.DefaultLifecycleObserver
+import androidx.lifecycle.LifecycleOwner
+import ch.twint.payment.sdk.Twint
+import ch.twint.payment.sdk.TwintPayResult
+import com.adyen.checkout.core.exception.CheckoutException
+
+object Twint {
+
+    private var twintObject: Twint? = null
+
+    private var onResultListener: ((TwintPayResult) -> Unit)? = null
+
+    fun initialize(activity: ComponentActivity) {
```

Comment:
Let's add some kdocs to this and the class 🙏

Reply:
Ahhh yes of course

Reply:
Added it, let me know if I could improve the docs